bundle/deploy/terraform/load.go

```diff
@@ -43,6 +45,10 @@ func (l *load) Apply(ctx context.Context, b *bundle.Bundle) error {
 		return err
 	}
 
+	if slices.Contains(l.modes, ReplaceResources) {
+		b.Config.Resources = config.Resources{}
```

Comment:
Why are we removing the resources fetched from the remote terraform state?  Maybe I'm missing something obvious but my understanding was that the purpose of this command is to get a snapshot of the deployed resources, to create a diff view on the VSCode side.